gr-uhd/lib/usrp_sink_impl.cc

```diff
@@ -615,14 +571,14 @@ namespace gr {
           // If it's on the first sample, immediately do the tune:
           GR_LOG_DEBUG(d_debug_logger, boost::format("Received tx_freq on start of burst."));
           pmt::pmt_t freq_cmd = pmt::make_dict();
-          freq_cmd = pmt::dict_add(freq_cmd, cmd_freq_key(), value);
+          freq_cmd = pmt::dict_add(freq_cmd, pmt::mp("freq"), value);
```

Comment:
Using "pmt::dict_add()" with precomputed `pmt::mp` is highly desirable, because the string hashing and symbol table lookups done every time you call `pmt::mp("same string as before")` can become a problem. That's why I removed these ad-hoc `mp` / `pmt::intern` calls on master ;)